pyproject.toml

```diff
@@ -113,6 +113,37 @@ include = [
     "/cosmos",
 ]
 
+######################################
+# TESTING
+######################################
+
+[tool.hatch.envs.tests]
+dependencies = [
+    "pytest>=6.0",
+    "pytest-split",
+    "pytest-dotenv",
+    "requests-mock",
+    "pytest-cov",
+    "pytest-describe",
+    "types-requests",
+    "mypy",
+    "sqlalchemy-stubs", # Change when sqlalchemy is upgraded https://docs.sqlalchemy.org/en/14/orm/extensions/mypy.html
+
+    "apache-airflow-providers-docker>=3.5.0",
+    "apache-airflow-providers-cncf-kubernetes>=5.1.1",
+]
+
+[[tool.hatch.envs.tests.matrix]]
+python = ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
@jlaneve, were you able to identify a hatch native way for us to test against multiple versions of Airflow? Or should we follow, as part of a separate PR, the `nox` approach we adopted in the Databricks providers package (https://github.com/astronomer/astro-provider-databricks)?

Reply:
yes - this now works locally and in the actions! see the `pyproject.toml` file and the `.github/workflows/test.yaml` files, respectively.